### ISSUE TITLE
Labels

### ISSUE THREAD

**Issue by zanderson9** [vincenzopalazzo/material-ui-swing#55]
Vincent, thank you for addressing disabled labels issue in the most recent library update as of May 27, 2019.  With those updates, we are now able to control the appearance of the disabled text.
But, at the same time, we lost 2 important features with the labels: 1) ability to set custom font on a label, and 2) ability to set custom color in a label.

The attached document illustrates what is happening with the labels. So even though we need the ability to display disabled text, but we could not afford to lose those font and color features. So we decided to roll back the library to May 21 release.

In summary, we need to have all 3 features available: 1) disable label visibility; 2) label's response to custom font setting in our code; 3) label's response to custom color setting in our code.

Please research what caused features 2) and 3)  to disappear in the latest release?

[LabelsIssue.pdf](https://github.com/vincenzopalazzo/material-ui-swing/files/3230358/LabelsIssue.pdf)

**Comment by vincenzopalazzo** [comment]
Hi @zanderson9,

The problem is this code:

https://github.com/vincenzopalazzo/material-ui-swing/blob/c4c0efeb77cc8d29666188f62dce9567e71c238c/src/mdlaf/components/label/MaterialLabelUI.java#L38-L56

The component does repaint with the setting default, but I try to listen to propriety change, this problem is the same as the button.

If I resolve this problem I should add a new Style to the default button

**Comment by vincenzopalazzo** [comment]
Hi @zanderson9 into this commit I resolved this problem
